app/processing/process-payment-request.js

```diff
@@ -17,18 +19,27 @@ const processPaymentRequest = async (scheduledPaymentRequest) => {
   }
 
   const paymentRequests = await transformPaymentRequest(paymentRequest)
+
+  // if FRN is closed, remove AR
+  const agreementIsClosed = config.handleSchemeClosures ? await isAgreementClosed(paymentRequest) : false
+  if (agreementIsClosed) {
+    paymentRequests.completedPaymentRequests = paymentRequests.completedPaymentRequests.filter(paymentRequest => paymentRequest.ledger === 'AP')

```

Comment:
Suggest using the existing AP constant, rather than just a string here.

Also might be best to encapsulate this in it's own function and file so it's easier to write targetted tests against.

Reply:
Process moved out to separate file and using AP constant instead of string